plugins/fleet.yaml

```diff
@@ -34,8 +38,10 @@ spec:
     files:
     - from: "/fleet.exe"
       to: "."
+    - from: "./LICENSE"
+      to: "."
     bin: "fleet.exe"
-  shortDescription: A cluster fleet info plugin
+  shortDescription: Shows what is going on in a fleet of clusters (config & resources)
```

Comment:
Can you squeeze that below ~55 characters? Above that it gets truncated in `krew search`.

Reply:
Sure will do and thanks for taking the time to provided feedback, appreciate it! Out of curiosity, is the 55 char limit documented somewhere, did I miss something?